keyboards/handwired/pterodactyl/twimaster.c

```diff
@@ -0,0 +1,207 @@
+/*************************************************************************
```

Comment:
This should be removed, in favor of `i2c_master.c` (located in `/drivers/(ARCH)/i2c_master.[ch]`).

This may also require that some of the i2c commands being changed.